pallets/streaming/src/lib.rs

```diff
@@ -55,7 +55,12 @@ type BalanceOf<T> =
     <<T as Config>::Assets as Inspect<<T as frame_system::Config>::AccountId>>::Balance;
 type AccountOf<T> = <T as frame_system::Config>::AccountId;
 
-// Struct for Stream
+#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum StreamStatus {
+    Ongoing,
+    Completed,
```

Comment:
why do we need this field? if there aren't any funds left then certainly it's Completed

Reply:
If combined with the loans pallet later, I think there will be more status, @i5possible what do you think of this?

Reply:
I am thinking of the Cancelled/AsCollatoral/Redeemed status. If we don't have the status now. We may need to have a data migration. Any other thoughts?

Reply:
If we cancel then the stream is removed actually so the status wont be useful